broker/broker.go

```diff
@@ -77,3 +82,7 @@ func (bkr *Broker) newLoggingSession(action string, data lager.Data) lager.Logge
 	logger.Info("start")
 	return logger
 }
+
+func (bkr *Broker) Cells() backend.Backends {
+	return bkr.scheduler.Backends
```

Comment:
How about returning `bkr.config.Backends` or even an `CellsApiStruct` instead of reaching into the scheduler and returning an object that is being used for behaviour and is better left private to the scheduler.